src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -217,3 +217,34 @@ import { ViewTransitions } from 'astro:transitions';
 :::note[known limitations]
 The `morph` animation cannot be simulated in traditional CSS. So any element using this animation will not be animated.
 :::
+
+## Caveats
+
+When using the `<ViewTransitions />` component page navigations do not behave exactly the same as they do when navigating directly to a page (such as typing the URL into the address bar).
+
+### Scripts
+
+When navigating between pages with the `<ViewTransitions />` component scripts are run in sequential order to match browser behavior. If you have code that sets up global state, this state will need to take into account that the script might execute more than once. You can usually do this by checking for the global state and conditionally executing your code.
+
+```astro
+<script is:inline>
+  if(!window.SomeGlobal) {
+    window.SomeGlobal = {} // ....
+  }
+</script>
+```
+
+Module scripts are only ever executed once, because the browser keeps track of which modules are already loaded. For these scripts you do not need to worry about re-execution.
```

Comment:
```suggestion
Module scripts are only ever executed once because the browser keeps track of which modules are already loaded. For these scripts, you do not need to worry about re-execution.
```
Commas before Yan gets here!